### ISSUE TITLE
Stream URL

### ISSUE THREAD

**Issue by iDevelopper** [agilie/DisPlayers-Audio-Visualizers#2]
Really nice project!

How could it works with stream URL (no file)?

Would be possible?

**Comment by kalamaznik** [comment]
Hi, we use local file "audio2.mp3" for Demo project.

**Comment by iDevelopper** [comment]
Yes I know, how could we use a stream URL instead of a file? (e.g. http://streaming.shoutcast.com/cooljazzflorida?lang=en-us )

**Comment by kalamaznik** [comment]
Sorry, we should use local files with following formats:

        @"aac",
        @"caf",
        @"aif",
        @"aiff",
        @"aifc",
        @"mp3",
        @"mp4",
        @"m4a",
        @"snd",
        @"au",
        @"sd2",
        @"wav"

**Comment by iDevelopper** [comment]
Thanks, yes I know. My question is relative to a new feature wich can allow streaming also live stream URL...
